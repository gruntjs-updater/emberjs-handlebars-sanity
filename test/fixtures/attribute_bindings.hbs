<div class="sidebar-section leg" {{bind-attr id="id"}} {{action 'highlightLeg'}}>
  <div class="col-xs-2">
    <div class="small-icon" {{bind-attr class="modeIconClass"}}></div>
  </div>
  <div class="col-xs-10">
    <p>
      {{#if mode}}
        {{formatTime departTime}}
        <br />
        <strong><a target="_blank" {{bind-attr href="timeTableUrl"}}>Bus {{routeCode}}</a></strong>
        towards {{destinationText}}
        <br />
        {{durationMinutes}} min {{stops}} - Stop no
      {{else}}
        <strong>Walk</strong> to {{destinationText}}
        <br />
        About {{durationMinutes}} min, {{walkDistanceMetres}} m
      {{/if}}
    </p>
  </div>
</div>
