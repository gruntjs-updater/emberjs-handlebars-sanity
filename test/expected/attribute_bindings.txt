Found template with duplicate attribute and attribute binding 'class'